st2client/st2client/models/core.py

```diff
@@ -547,6 +547,16 @@ def get_output(self, execution_id, output_type=None, **kwargs):
 
         return response.text
 
+    @add_auth_token_to_kwargs_from_env
+    def get_result(self, execution_id, **kwargs):
+        url = "/%s/%s/result" % (self.resource.get_url_path_name(), execution_id)
+
+        response = self.client.get(url, **kwargs)
+        if response.status_code != http_client.OK:
+            self.handle_error(response)
+
+        return orjson.loads(response.text)
```

Comment:
is `orjson.loads` needed here?  `get_output` just returns `response.text`

Reply:
@guzzijones from the built-in actions I checked, they return a json block {}, however the `response.text` would be the string version of that. 
If its expected that the consumer does the `loads()` on the response, then I can swap it to just return the string text. I figured since it was a json string, it was nicer just to return a json object that is ready to reference.

Reply:
However after looking at the openapi spec for that endpoint 
```
      responses:
        '200':
          description: Execution result
          schema:
            type: string
            # Response is raw text so we don't want to perform validation on it
            validate_response: false
```
Its noted it returns a string on purpose. So its probably better to just return the string and let the user convert as needed.